common/chart.ts

```diff
@@ -67,6 +67,120 @@ export const getReposStarData = async (
   });
 };
 
+export const getReposLogoUrl = async (
+  repos: string[],
+  token = ""
+): Promise<{ repo: string; logo: string }[]> => {
+  const repoLogos: { repo: string; logo: string }[] = [];
+
+  for (const repo of repos) {
+    try {
+      const logo = await api.getRepoLogoUrl(repo, token);
+      repoLogos.push({ repo, logo });
+    } catch (error: any) {
+      let message = "";
+      let status = 500;
+
+      if (error?.response?.status === 404) {
```

Comment:
Let's remove the switch-like handler for response error when fetching repo logoUrl.

Reply:
Actually this `getReposLogoUrl` method is not used anywhere. I've made getting both star records and logo url into one `getRepoData` method. So I can just remove this whole `getReposLogoUrl` method.

Reply:
Then let's remove it.